### PR TITLE
Sms service

### PR DESCRIPTION
**Pull request by MikulaJan**
Review NullReferenceException for CreateSmsService.SendSms.
+ removed doubled semicolon on 63

### CODE REVIEW

File: backend/Origam.Common/OrigamSettings.cs [hunk 1]
```diff
@@ -267,6 +269,9 @@ public override string ToString()
 
         [Category("Model Connection")]
         public bool CheckFileHashesAfterModelLoad { get; set; } = true;
+        
+        [Category("SmsService")] 
+        public string SmsService { get; set; } = "Origam.Sms.DebugSmsService, Origam.Sms";
```
Comment (washibana) [inline backend/Origam.Common/OrigamSettings.cs:274]: Origam.Sms library doesn't contain DebugSmsService, as it was discussed previously, we should implement somewhere FileSystemSmsService and use it as a default value here.

**Review by washibana — CHANGES_REQUESTED** [review]

File: backend/Origam.Workflow/Service Agents/SmsServiceAgent.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                        Parameters.Get<string>("from"),
+                        Parameters.Get<string>("to"),
+                        Parameters.Get<string>("body"));
+                    break;
+                default:
+                    throw new ArgumentOutOfRangeException(
+                        "MethodName", MethodName,
+                        ResourceUtils.GetString("InvalidMethodName"));
+            }
+            if (result == null) {
```
Comment (washibana) [inline backend/Origam.Workflow/Service Agents/SmsServiceAgent.cs:47]: This check should be in `CreateSmsService` and the error message should provide more information on the problem - info on assembly and class that is used for the invocation.

**Review by washibana — CHANGES_REQUESTED** [review]

File: backend/Origam.Common/OrigamSettings.cs (hunk 1 above)
Comment (MikulaJan) [reply backend/Origam.Common/OrigamSettings.cs]: True, i just renamed to FileSystemSmsService and this class was also added to  Origam.Sms project with testing method.

Comment (MikulaJan) [reply backend/Origam.Common/OrigamSettings.cs]: FileSystemSmsService will be separated repo and project.

**Review by washibana — APPROVED** [review]

**Comment by washibana** [comment]
@tvavrda Could you please review it as well?

File: backend/Origam.Common/OrigamSettings.cs [hunk 3]
```diff
@@ -267,6 +269,9 @@ public override string ToString()
 
         [Category("Model Connection")]
         public bool CheckFileHashesAfterModelLoad { get; set; } = true;
+        
+        [Category("SmsService")]
```
[the remark this answers is not in the document]
Comment (MikulaJan) [reply backend/Origam.Common/OrigamSettings.cs:273]: Renamed to SMS Service

**Pull request merged**